src/main/kotlin/routing/teamRouting.kt

```diff
@@ -0,0 +1,49 @@
+package routing
+
+import Stores.TeamObj
+import Stores.TeamStore
+import Stores.UserStore
+import Stores.UsersObj
+import com.sun.media.jfxmedia.logging.Logger
+import io.ktor.application.call
+import io.ktor.http.Parameters
+import io.ktor.locations.get
+import io.ktor.locations.post
+import io.ktor.request.receive
+import io.ktor.response.respond
+import io.ktor.routing.Route
+
+data class ListResponse<T>(val status: String, val reason: String, val data: List<T>?)
+data class CreateResponse(val status: String, val reason: String, val id: String?)
+
+object teamRouting{
```

Comment:
Capitalize the "t" in team. Also change the file name to be that as well. To change it easily you can have the carret over it and pres fn+f6